source/main/gameplay/RoRFrameListener.cpp

```diff
@@ -1512,6 +1504,12 @@ bool RoRFrameListener::frameStarted(const FrameEvent& evt)
 				}
 			}
 		}
+
+		if (!m_is_sim_paused)
+		{
+			BeamFactory::getSingleton().updateFlexbodiesFinal();   // Waits until all flexbody tasks are finished 
+			BeamFactory::getSingleton().calcPhysics(dt);           // we simulate one truck, it will take care of the others (except networked ones)

```

Comment:
`// we simulate one truck, it will take care of the others (except networked ones)`
AFAIK this is a deprecated comment. It applied in the `Beam::frameStep()` era, but not anymore.